src/components/MyAccount/index.tsx

```diff
@@ -423,7 +454,31 @@ const MyAccountContent = ({
             </Grid>
             <Grid
               item
-              xs={isMobile ? 12 : 4}
+              xs={isMobile ? 12 : 3}
+              sx={{ '&.MuiGrid-item': { paddingTop: theme.spacing(isMobile ? 3 : 2) } }}
+            >
+              {edit ? (
+                <RegionSelector
+                  selectedCountryCode={countryCodeSelected}
+                  onChangeCountryCode={setCountryCodeSelected}
+                  horizontalLayout
```

Comment:
Removing this will give you more width (counterintuitive, region selector needs some refactor/cleanup later).
I think the language and preferred-weight dropdowns should also be full-width, something FE can fix out of band of this PR.

Reply:
That did the trick. Thanks!